test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -164,7 +164,7 @@ public async Task CustomTextFormat()
                 var response = await client.GetAsync("/api/values/2");
                 response.EnsureSuccessStatusCode(); // Status Code 200-299
 
-                WaitForProcessorInvocations(spanProcessor, 4);
+                WaitForProcessorInvocations(spanProcessor, 2);
```

Comment:
I'm probably just missing it... but what accounted for this change?

Reply:
Hi @alanwest , looking at that file, if you check the function it wasn't using the parameter (it was hardcoded 2). So, i changed from 4 to 2 to use the parameter. But, this isn't releated to the metapackages at all.

Reply:
Ah, yes that makes sense. 👍